USAGE.rst

```diff
@@ -0,0 +1,137 @@
+kafkacat
+========
+
+kafkacat is a command line tool for working with Kafka. Described as "netcat for Kafka", it is a swiss-army knife of tools for inspecting and creating data in Kafka.
+
+If you already know ``kafka-console-producer`` and ``kafka-console-consumer`` then it's like those, on steroids.
+
+kafkacat is an open-source tool, available at https://github.com/edenhill/kafkacat. It is not included in Confluent Platform.
+
+``kafkacat`` the Consumer
+-------------------------
+
+Let's start simple. Give ``kafkacat`` a broker and a topic, and you'll see its contents:
```

Comment:
avoid using `we`, `lets` (see https://confluentinc.atlassian.net/wiki/spaces/DOC/pages/161743785/Style+Guide+for+Confluent+Documentation#StyleGuideforConfluentDocumentation-VoiceandTone). i'd remove `Let's start simple.`